tests/test_diff.py

```diff
@@ -268,6 +278,146 @@ def test_annotation_comparison(self):
         self.assertEqual(2, len(unmatched), unmatched)
         self.assertEqual(0, len(errors), errors)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_skeleton_annotation_comparison(self):
+        categories = {
+            AnnotationType.label: LabelCategories.from_iterable(
+                [
+                    "skeleton",
+                    ("point1", "skeleton"),
+                    ("point2", "skeleton"),
+                    ("point3", "skeleton"),
+                ]
+            ),
+            AnnotationType.points: PointsCategories.from_iterable(
+                [
+                    (0, ["point1", "point2", "point3"], set()),
+                ]
+            ),
+        }
+        a = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id=1,
+                    annotations=[
+                        Skeleton(
+                            [
+                                Points([0, 1], [2]),
```

Comment:
Could you use symbolic visibility constants? Without them, it's not very clear what's going on.

Reply:
Used them in this test